chirp/drivers/baofeng_uv17.py

```diff
@@ -65,27 +64,33 @@ def _download(radio):
     radio.status_fn(status)
 
     for block_number in radio.BLOCK_ORDER:
-        block_index = memory_map.index(block_number) + 1
-        start_addr = block_index * 0x1000
-        for addr in range(start_addr, start_addr + 0x1000,
-                          radio.BLOCK_SIZE):
-            frame = radio._make_read_frame(addr, radio.BLOCK_SIZE)
-            # DEBUG
-            LOG.debug("Frame=" + util.hexprint(frame))
+        if block_number not in memory_map:
+            """Memory block not found."""
```

Comment:
Probably better to make this a comment. As a string it gets processed and stored in memory for no reason.

Reply:
Done